src/common/notfound/NotFound.tsx

```diff
@@ -0,0 +1,49 @@
+import { Container, Heading, Flex, Text, Divider } from '@chakra-ui/layout';
+import { Button, Box } from '@chakra-ui/react';
+import { HiOutlineCalendar } from 'react-icons/hi';
+import { Link } from 'react-router-dom';
+
+import { useDarkMode } from 'lib/hooks/useDarkMode';
+import { getReadableTerm } from 'lib/utils/terms';
+
+type Props = {
+  item: string;
+  term: string;
+  timetableButton?: boolean;
+  timetable?: boolean;
+};
+
+export function NotFound({ item, term, timetable }: Props) {
+  const mode = useDarkMode();
+  var display: string = '';
+  if (timetable) display = ' timetable';
```

Comment:
Not sure I'm a fan of this. I think we should just have the full string passed in in `children` (what is currently `item`) and only conditionally render the button based on the prop

Reply:
Would it be best to change the wording so that the term comes at the end of the string? Its just a bit more wordy
<img width="500" alt="Screen Shot 2022-05-12 at 9 15 38 AM" src="https://user-images.githubusercontent.com/77510623/168121474-2aca4006-b1c1-4f6d-94dd-33fedc09e8e6.png">
.